src/helpers/formatting/formatDate.js

```diff
@@ -0,0 +1,25 @@
+function formatDate(date, config = {}) {
+    const { day = "2-digit", month = "2-digit", year = "2-digit", separator = ".", locale = "en-GB" } = config;
+    const GENERAL_ERROR_MESSAGE = "Invalid date input";
+
+    try {
+        if (!date) {
+            throw new Error(GENERAL_ERROR_MESSAGE);
+        }
+
+        const dateObject = new Date(date);
+
+        if (isNaN(dateObject.getTime())) {
```

Comment:
maybe we don't have to do this, we can just check whether dateObj is undefined or not